src/test/rbd_mirror/test_Instances.cc

```diff
@@ -131,6 +133,7 @@ TEST_F(TestInstances, NotifyRemove)
   ASSERT_EQ(0, on_init.wait());
 
   instances.acked({instance_id1, instance_id2});
+  usleep(25000);
```

Comment:
Nit: this is racy -- the real cause of the issue needs to be fixed.

Reply:
I think this is related to the test: rbd_mirror code[0] calls part of the test[1] after unlocking a mutex. Also correct me if I'm wrong, `usleep` also is used in the test below in line#149 for a similar reason?

[0] https://github.com/ceph/ceph/blob/master/src/tools/rbd_mirror/Instances.cc#L147-L149
[1] https://github.com/ceph/ceph/blob/master/src/test/rbd_mirror/test_Instances.cc#L49-L51

Reply:
Your change only sleeps once which just seems racy -- the other case sleeps up to 100 times (25 seconds)